content/docs/setup/kubernetes/upgrade/index.md

```diff
@@ -94,6 +71,31 @@ including configurations generated using
 The rolling update process will upgrade all deployments and configmaps to the new version. After this process finishes,
 your Istio control plane should be updated to the new version. Your existing application should continue to work without
 any change. If there is any critical issue with the new control plane, you can rollback the changes by applying the yaml files from the old version.
+{{% /tab %}}
+
+{{% tab name="Helm upgrade" cookie-value="helmupgrade" %}}
+If you installed Istio using [Helm and Tiller](/docs/setup/kubernetes/install/helm/#option-2-install-with-helm-and-tiller-via-helm-install),
+the preferred upgrade option is to let Helm take care of the upgrade:
+
+1. To keep updated all the Istio [Custom Resource Definitions](https://kubernetes.io/docs/concepts/extend-kubernetes/api-extension/custom-resources/#customresourcedefinitions) (CRDs), you must upgrade the `istio-init` chart. If the chart doesn't exist you must install the new release.
+
+    {{< text bash >}}
+    $ helm upgrade --install istio-init install/kubernetes/helm/istio-init --namespace istio-system
+    {{< /text >}}
+
+1. Check that all the CRD creation jobs completed successfully to verify that the Kubernetes API server received all the CRDs:
+
+    {{< text bash >}}
```

Comment:
need a boilierplate for these checks.  I'll try to tackle that post-release and we can backport.

Reply:
I tried this, but boilerplate in numbered lists didn't work. It disabled all copy buttons from that point on in the doc. Martin has some fixes in #3706 that may fix it, if you want to try again.
```
1. {{< boilerplate check-crd-count >}}
```
check-crd-count.md
```
Verify that all `58` Istio CRDs were committed to the Kubernetes api-server using the following command:

    {{< text bash >}}
    $ kubectl get crds | grep 'istio.io\|certmanager.k8s.io' | wc -l
    58
    {{< /text >}}
```

Reply:
thanks @frankbu.  I'll give that a go post release after a rebase on top of geeknoid's site infra changes.

Reply:
I think boilerplates are fully composable with lists now.